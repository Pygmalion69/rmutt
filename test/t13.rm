// complex mapping syntax
a: {"a"|"b"}>{"a"%{"0"|"zero"} "b"%{"1"|"one"}};