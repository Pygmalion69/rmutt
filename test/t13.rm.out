one
zero
1
0
